src/foam/u2/view/TableSummaryView.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.view',
+  name: 'TableSummaryView',
```

Comment:
SummaryView has another meaning. It is a synonym of CitationView. Can you think of a better name?

Reply:
EmbeddedTableView, InlineTableView ?

Reply:
I like EmbeddedTableView, InlineTableView is too similar to InlineBrowserView